src/redux/services/fileMonitor.ts

```diff
@@ -15,55 +15,57 @@ let watcher: FSWatcher;
 
 export function monitorRootFolder(folder: string, appConfig: AppConfig, dispatch: AppDispatch) {
   if (watcher) {
-    watcher.close();
-  }
-
-  watcher = watch(folder, {
-    ignored: appConfig.scanExcludes,
-    ignoreInitial: true,
-    persistent: true,
-    usePolling: true,
-    interval: 1000,
-  });
+    const watched = Object.keys(watcher.getWatched());
```

Comment:
here if the watcher exists we should update the `ignored` property to the most up to date `appConfig.scanExcludes`

Reply:
good catch - unfortunately there doesn't seem to be a way to change the ignored property after creation - i'll go back to recreating the watcher for now